wrappers/src/sync_user_cs.cpp

```diff
@@ -179,31 +179,31 @@ extern "C" {
             switch (profile_field)
             {
             case UserProfileField::name:
-                field = user->user_profile().name;
+                field = user->user_profile().name();
```

Comment:
Was this not working before?

Reply:
I think that's due to https://github.com/realm/realm-core/pull/4244 changing the Core API.

Reply:
This PR needed the newest Core version to use the new `write_copy` released with https://github.com/realm/realm-core/pull/4827.

https://github.com/realm/realm-core/pull/4244 is part of this release which includes changing the properties to functions, see https://github.com/realm/realm-core/pull/4244/files#diff-a3fd6f3088dd3faa41d4cbffd668496ceb272c4937cdce9fc4919a9acbc4a1a7R77.